scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -31,25 +31,85 @@
 
 # hardcode the upgrade you want to do here.
 def get_upgrade_set():
-    state_exp_settings = StateExpirationSettings(max_entry_expiration=Uint32(6312000),
-                                                 min_temp_entry_expiration=Uint32(
-                                                     16),
-                                                 min_persistent_entry_expiration=Uint32(
-                                                     86400),
+
+    max_contract_size = Uint32(65536) # 64 kb
+
+    contract_size_upgrade_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_MAX_SIZE_BYTES,
+        contract_max_size_bytes = max_contract_size)   
+        
+    compute_settings = ConfigSettingContractComputeV0(ledger_max_instructions=Int64(1000000000),
+                                                 tx_max_instructions=Int64(100000000),
+                                                 fee_rate_per_instructions_increment=Int64(100),
+                                                 tx_memory_limit=Uint32(41943040)) # 40 mb
+
+    compute_upgrade_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_COMPUTE_V0,
+        contract_compute = compute_settings)    
+    
+    contract_ledger_cost_settings = ConfigSettingContractLedgerCostV0(ledger_max_read_ledger_entries=Uint32(400),
+                                                 ledger_max_read_bytes=Uint32(2048000),
+                                                 ledger_max_write_ledger_entries=Uint32(200),
+                                                 ledger_max_write_bytes=Uint32(1024000),
+                                                 tx_max_read_ledger_entries=Uint32(30),
+                                                 tx_max_read_bytes=Uint32(133120), # 130 kb
+                                                 tx_max_write_ledger_entries=Uint32(20),
+                                                 tx_max_write_bytes=Uint32(66560), # 65 kb 
+                                                 fee_read_ledger_entry=Int64(20000),
+                                                 fee_write_ledger_entry=Int64(30000),
+                                                 fee_read1_kb=Int64(1000),
+                                                 fee_write1_kb=Int64(1000),
+                                                 bucket_list_size_bytes=Int64(32212254720), # 30 * 1024 * 1024 * 1024 for 30 GB
```

Comment:
The target in the doc is 18GB. Also for futurenet/testnet we probably should go for 2GB (as the initial size is just 0, unlike mainnet where initial size is ~16GB)